### ISSUE TITLE
[BUG] weird scaling numbers when training nuclei model

### ISSUE THREAD

**Issue by pawlowska** [MouseLand/cellpose#786]
**Describe the bug**
Different diameters displayed when training from nuclei model; when using the model I also need to use an artificial diameter. 

My nuclei have diameter of 50-60 and this number works when segmenting in GUI with nuclei model. I then corrected the masks manually and wanted to use them for further training. The manual says that for training the nuclei model, cells need to have diameter 17, so I downscaled my images and masks. I then did the training. However, when I load the model, the default diameter displayed in GUI is 3.164 and not 17. When I put in 50 or 60, I do not get meaningful results either. For the trained model to work, I need to put in diameter of around 12.

Then, during training I see displayed as below. How to understand that? Are the images rescaled by cellpose, or do I need to rescale them? 

And mostly, what is the meaning of `>>>> model diam_labels =  3.164 (mean diameter of training ROIs)` and is this connected to having to put in 12 in GUI instead of 50?

To conclude, either I'm missing something or there is a bug in scaling when training from nuclei.

**Screenshots**
![cellpose training scaling](https://github.com/MouseLand/cellpose/assets/11158300/c5cfaf9b-492e-4545-9666-54a808e1bc85)

**Comment by mrariden** [comment]
The line you referenced means that the mean diameter of the training masks (the input masks during training) was 3.164 pixels. Can you double check that your input masks are actually 17 pixels in diameter and not closer to 3? You can also attach an example image if you put it into a zip archive. 

Also, please provide the entire command used for training and the output.

**Comment by mrariden** [comment]
closing due to inactivity, reopen if necessary

**Comment by pawlowska** [comment]
Hi @mrariden I apologize for missing your reply. I'm attaching one image&mask pair of the 46 used. 
[seq_rescaled_05_example.zip](https://github.com/MouseLand/cellpose/files/13378675/seq_rescaled_05_example.zip)
command:
` python -m cellpose --train --dir ./seq_rescaled_05/ --verbose --use_gpu --pretrained_model nuclei`

**Comment by carsen-stringer** [comment]
I think something weird must have happened when you downsampled your images, this is not necessary during training. please try to train with the full images, cellpose will downsample them automatically during training. I will reopen if this doesn't work

**Comment by pawlowska** [comment]
Thank you for your reply! I wonder if you have an example somewhere on how to prepare 3D training data including dealing with anisotropy. I tried asking in forum.image.sc as well but no luck: https://forum.image.sc/t/training-cellpose-on-3d-data-with-anisotropy/85434

In the dataset above I did rescaling for convenience because the original dataset had high magnification/small pixel size for the purpose of capturing some cellular details, much too high for the purpose of nuclei segmentation.

**Comment by carsen-stringer** [comment]
ah it's a good point, I've just added an anisotropy flag to the make_train.py function, described here: https://cellpose.readthedocs.io/en/latest/train.html#re-training-a-model

you can use the anisotropy flag so that objects are scaled correctly in ZY and ZX, you can also train separate models for ZY+ZX and for YX, I will add code for this soon.

**Comment by pawlowska** [comment]
Thank you! I expect that the correct approach will partly depend on objective NA as it determines the ratio between lateral and axial resolution (for low NA the orthogonal projections "look different" so it makes sense that it should not be the same model!), but that is a whole separate discussion.
